### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by ibevers**
Updated with links to updated activities in:
https://raw.githubusercontent.com/ReproNim/reproschema-library/enh/rc4/activities/

The conditional logic of what activities are shown based on age conforms to:
https://nda.nih.gov/nda/nimh-common-data-elements

Demo:
https://ibevers.github.io/reproschema-ui/#/?url=https://raw.githubusercontent.com/ibevers/nimh-minimal/update_schema/nimh_minimal/nimh_minimal_schema

### CODE REVIEW

**Comment by satra** [comment]
this added the .venv directory

**Comment by ibevers** [comment]
> this added the .venv directory

Fixed @satra

File: nimh_minimal/nimh_minimal_schema [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "@context": [
         "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc4/contexts/generic",
         {
-            "activity_path": "https://raw.githubusercontent.com/ReproNim/reproschema-library/80867e36fb2c00563290486bf3f3bbeb3198f5cb/activities/"
+            "activity_path": "https://raw.githubusercontent.com/ReproNim/reproschema-library/enh/rc4/activities/"
```
Comment (satra) [inline nimh_minimal/nimh_minimal_schema]: shall we switch this to a specific committish now, if this all works?

**Comment by satra** [comment]
also can we update the github workflow checksum, so this protocol renders using the correct ui committish?

File: nimh_minimal/nimh_minimal_schema (hunk 1 above)
Comment (ibevers) [reply nimh_minimal/nimh_minimal_schema]: I realized there is a test case on enh/rc4 in reproschema-library that wasn't passing--should be fixed when this is merged:
https://github.com/ReproNim/reproschema-library/pull/80

I ran the test locally before pushing.

**Comment by ibevers** [comment]
> also can we update the github workflow checksum, so this protocol renders using the correct ui committish?

Slightly confused by what you mean here. My conclusion from the conversation about the UI before is that the only way to render a protocol with a specific version of the UI is to host that version somewhere. Would you mind explaining a bit more?

**Comment by satra** [comment]
> My conclusion from the conversation about the UI before is that the only way to render a protocol with a specific version of the UI is to host that version somewhere.

indeed and it would be hosted here in this repo. so updating the checksum in config.env should be all that's needed.

**Comment by ibevers** [comment]
But what should I update the checksum to? The latest checksum for the fix/static branch of reproschema-ui?

https://github.com/ReproNim/reproschema-ui/tree/fix/static

**Comment by satra** [comment]
i think that was already merged, so the current checksum should be fine.

**Comment by satra** [comment]
sorry it wasn't merged. so yes, you can take the committish of that branch and use it.

**Comment by ibevers** [comment]
> sorry it wasn't merged. so yes, you can take the committish of that branch and use it.

Done @satra

File: nimh_minimal/nimh_minimal_schema [hunk 2]
```diff
@@ -96,15 +112,17 @@
...
-            "activity_path:RCADS-25/RCADS-25_schema",
+            "activity_path:RCADS-25-C/RCADS25_caregiver_administered_schema",
+            "activity_path:RCADS-25-Y/RCADS25_youth_administered_schema",
             "activity_path:ThankYou/ThankYou_schema"
         ],
         "shuffle": false,
         "allow": [
             "reproschema:AllowExport",
-	    "reproschema:AutoAdvance"
+	        "reproschema:AutoAdvance"
```
Comment (satra) [inline nimh_minimal/nimh_minimal_schema]:
```suggestion
	    "reproschema:AutoAdvance"
```

File: nimh_minimal/nimh_minimal_schema [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "@context": [
         "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc4/contexts/generic",
         {
-            "activity_path": "https://raw.githubusercontent.com/ReproNim/reproschema-library/80867e36fb2c00563290486bf3f3bbeb3198f5cb/activities/"
+            "activity_path": "https://raw.githubusercontent.com/ReproNim/reproschema-library/enh/rc4/a23a13875c7262c0bd0d77bd90c1ec296c6d1116/activities/"
```
Comment (satra) [inline nimh_minimal/nimh_minimal_schema:5]: i merged this, but i think this is incorrect. the branch name should not be there.

Comment (ibevers) [reply nimh_minimal/nimh_minimal_schema:5]: Oops, I will double-check next time @satra. I created a new PR: https://github.com/ReproNim/nimh-minimal/pull/6

**Pull request merged**